packages/services/src/manager.ts

```diff
@@ -199,5 +199,10 @@ namespace ServiceManager {
      * The server settings of the manager.
      */
     readonly serverSettings?: ServerConnection.ISettings;
+
+    /**
+     * The default drive for the contents manager.
+     */
+    readonly defaultDrive?: Contents.IDrive;
```

Comment:
Not your fault, but the `new Drive()` call in the constructor should probably get passed `serverSettings`, if provided here.